### ISSUE TITLE
Netlink events for MP_TCPRST

### ISSUE THREAD

**Issue by matttbe** [multipath-tcp/mptcp_net-next#188]
Linked to #53 : we would like to have netlink events in case of MP_TCPRST and visible with `ip monitor`

**Comment by dcaratti** [comment]
I can have a look at this, while re-validating the MP_RESET packetdrill tests (see #143)

**Comment by dcaratti** [comment]
according to this test,

```
# ip mptcp monitor &
# ../../packetdrill/packetdrill  -vvvv mp_reset_single.pkt 
socket syscall: 1623086537.472101
setsockopt syscall: 1623086537.472434
bind syscall: 1623086537.472591
listen syscall: 1623086537.472632
inbound injected packet:  0.001394 S 0:0(0) win 32792 <mss 1460,sackOK,nop,nop,nop,wscale 7,mp_capable v1 flags: |H| >
outbound sniffed packet:  0.002059 S. 4256084969:4256084969(0) ack 1 win 65535 <mss 1460,nop,nop,sackOK,nop,wscale 8,mp_capable v1 flags: |H| sender_key: 4370654907521782500>
inbound injected packet:  0.015536 . 1:1(0) ack 4256084970 win 256 <mp_capable v1 flags: |H| sender_key: 2 receiver_key: 4370654907521782500>
accept syscall: 1623086537.489213
inbound injected packet:  0.017994 R. 1002:1002(0) ack 4256084969 win 256 
outbound sniffed packet:  0.018090 . 4256084970:4256084970(0) ack 1 win 256 <dss dack4 3007449509 flags: A>
inbound injected packet:  0.018498 . 1:1(0) ack 4256084969 win 256 <mp_reset 1>
write syscall: 1623086537.490965
outbound sniffed packet:  0.019680 P. 4256084970:4256085970(1000) ack 1 win 256 <dss dack4 3007449509 dsn8 11412280727116408234 ssn 1 dll 1000 no_checksum flags: MmA,nop,nop>
inbound injected packet:  0.022236 P. 1:1(0) ack 4256085970 win 256 <dss dack4 1087334802 dsn8 13263177308786788773 ssn 1 dll 0 no_checksum flags: MmA,nop,nop>
inbound injected packet:  0.022431 R. 1:1(0) ack 4256084969 win 256 <mp_reset 1>
inbound injected packet:  0.022586 . 1:1(0) ack 4256084969 win 256 
outbound sniffed packet:  0.022788 R 4256084969:4256084969(0) win 0 
[       CREATED] token=cbfe3c23 remid=0 locid=0 saddr4=192.168.178.118 daddr4=192.0.2.1 sport=8080 dport=54023
[   ESTABLISHED] token=cbfe3c23 remid=0 locid=0 saddr4=192.168.178.118 daddr4=192.0.2.1 sport=8080 dport=54023
[     SF_CLOSED] token=cbfe3c23 remid=0 locid=0 saddr4=192.168.178.118 daddr4=192.0.2.1 sport=8080 dport=54023 backup=0 reset_reason=1 reset_flags=0x0
[        CLOSED] token=cbfe3c23
# 
```
I think this ticket can be closed as-is. @matttbe correct?

**Comment by matttbe** [comment]
@dcaratti good point, I missed that from: https://git.kernel.org/pub/scm/network/iproute2/iproute2-next.git/commit/?id=ff619e4fd3706f064a0325fcc70f811fb3027b85

Closing then :)
